Data/TestFolder/sample.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Can probably remove this `sample.json` file based on the discussions we've had about serializing/deserializing json data in the last few days. This file and its general structure is probably invalid now.